### PR TITLE
WMS-800 | Update ListView style

### PR DESCRIPTION
**Pull request by kress95**
#### :thinking: What?

Update ListView to be more consistent with [figma](https://www.figma.com/file/DTyoSveNksHpZCdU8a7BNs/Dashboard?node-id=104%3A1804):

image

#### :mag: How it looks?

image

#### :man_shrugging: Why?

It's part of the problem-solving flow.

#### :pushpin: Jira Issue

[WMS-800](https://paacklogistics.atlassian.net/browse/WMS-800)

### :fire: Extra

- I'll Include the input icon in the next PR.

### CODE REVIEW

**Comment by renanpvaz** [comment]
Few things first: 
- the card could've been created in the design system board
- the figma link in these cases could've been available in the description or the card at least
- a picture of what it looks like now would be nice

**Review by renanpvaz — APPROVED** [review]
oops, I understood from the title that this was going to be a full update including the new features for the list view seen in figma

**Comment by PedroHLC** [comment]
Also, could you add the headers to the showcase? There isn't a ListView page, but the Layout shows one: http://localhost:1234/#Basics/Layouts/SplitSelectable

**Review by PedroHLC — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -44,5 +44,8 @@
   "devDependencies": {
     "elm-doc-preview": "^5.0.5",
     "elm-review": "^2.4.4"
+  },
+  "resolutions": {
```
Comment (PedroHLC) [inline package.json:48]: We won't need this if we merge #192  first.
But it's also an option to merge this one first and add a commit to #192 removing this.

**Pull request merged**